tests/ApiTransformerTest.php

```diff
@@ -93,4 +93,62 @@ public function test_output_transforming()
 
         $this->assertEquals($expectedOutput, $transformedOutput);
     }
-}
\ No newline at end of file
+
+    public function test_the_datatype_is_nullable()
+    {
+        $this->transformer->setApiMapping([
+            'price' => ['newName' => 'price', 'dataType' => 'nullable|int']
+        ]);
+
+        $input = [
+            'price' => '100'
+        ];
+
+        $expectedOutput = [
+            'price' => 100
+        ];
+
+        $this->assertSame($expectedOutput, $this->transformer->transformOutput($input));
+
+        $input = [
+            'price' => 0
+        ];
+
+        $expectedOutput = [
+            'price' => null
+        ];
+
+        $this->assertSame($expectedOutput, $this->transformer->transformOutput($input));
+
+        $this->transformer->setApiMapping([
+            'description' => ['newName' => 'description', 'dataType' => 'array|nullable']
+        ]);
+
+        $input = [
+            'description' => []
+        ];
+
+        $expectedOutput = [
+            'description' => null
+        ];
+
+        $this->assertSame($expectedOutput, $this->transformer->transformOutput($input));
+    }
+
+    public function test_arguments_can_be_passed_to_the_datatype()
+    {
+        $this->transformer->setApiMapping([
+            'price' => ['newName' => 'price', 'dataType' => 'float:2']
+        ]);
+
+        $input = [
+            'price' => '100.54'
```

Comment:
Change the input to prove that the output has 2 decimals. 
Either have 3 or 1.

Reply:
You have a good point here. I'll change that asap.